pkg/commands/sso/root.go

```diff
@@ -92,7 +109,12 @@ func (c *RootCommand) Exec(in io.Reader, out io.Writer) error {
 	// that another session (from a different profile) doesn't cause unexpected
 	// errors for the user flow. This forces a re-auth.
 	if c.InvokedFromProfileCreate || c.InvokedFromProfileUpdate || ForceReAuth {
+		c.Globals.AuthServer.SetParam("prompt", "login select_account")
+	}
+	if c.InvokedFromProfileSwitch {
 		c.Globals.AuthServer.SetParam("prompt", "login")
+		c.Globals.AuthServer.SetParam("login_hint", c.ProfileSwitchEmail)
+		c.Globals.AuthServer.SetParam("account_hint", c.ProfileSwitchCustomerID)
```

Comment:
Maybe we should check whether these are empty before setting? Asusming in the tranistion there may be a possibility they may be empty. However, I don't think there is a harm in sending empty values.